### PR TITLE
Enable password to be passed in via environment

### PR DESCRIPTION
**Pull request by delapsley**
* Enables user to pass ACI password in via the
  ACC_PROVISION_PASS environment variable.

### CODE REVIEW

File: provision/README.md [hunk 1]
```diff
@@ -22,5 +22,6 @@ optional arguments:
   -u , --username   APIC admin username to use for APIC API access
   -p , --password   APIC admin password to use for APIC API access
   -v, --verbose     Enable debug
-
+  
+Passw
```
Comment (readams) [inline provision/README.md]: Looks like some extraneous content here.

File: provision/acc_provision/acc_provision.py [hunk 2]
```diff
@@ -911,8 +912,12 @@ def provision(args, apic_file, no_random):
...
     if args.username:
         config["aci_config"]["apic_login"]["username"] = args.username
-    if args.password:
-        config["aci_config"]["apic_login"]["password"] = args.password
+
+    try:
+        config["aci_config"]["apic_login"]["password"] = os.environ['ACC_PROVISION_PASS']
+    except KeyError:
+        if args.password:
+            config["aci_config"]["apic_login"]["password"] = args.password
```
Comment (readams) [inline provision/acc_provision/acc_provision.py]: This whole chunk of code can simplified to:

    config["aci_config"]["apic_login"]["password"] = \
            os.environ.get('ACC_PROVISION_PASS', args.password)

Comment (mandeepdhami) [reply provision/acc_provision/acc_provision.py]: Should the command line parameter not override the one  in environment (that was a problem with the original patch as well)? I would recommend that it be something like:
'''
config["aci_config"]["apic_login"]["password"] = \
    args.password if args.password else os.environ.get('ACC_PROVISION_PASS')
'''

Comment (delapsley) [reply provision/acc_provision/acc_provision.py]: Much nicer thanks! Update following shortly.

**Comment by delapsley** [comment]
Thanks @readams. I've updated the comment in the README.

**Pull request merged**